protocol/transport/filters_manager.go

```diff
@@ -158,6 +158,8 @@ func (f *FiltersManager) InitCommunityFilters(pks []*ecdsa.PrivateKey) ([]*Filte
 
 		f.filters[filterID] = filter
 
+		f.logger.Debug("registering filter for", zap.String("chatID", filterID), zap.String("topic", rawFilter.Topic.String()))
```

Comment:
We could add a similar logs below when we add other filters

Reply:
Sounds good. I'll add them.